bin/yugabyted

```diff
@@ -566,6 +584,28 @@ class ControlScript(object):
                 (_, was_started) = self.maybe_start_yw(is_first_run, is_first_install)
                 should_callhome = should_callhome or was_started
 
+            if is_first_install and not join_ip:
+                if self.get_failed_node_processes():
+                    Output.log_error_and_exit(
+                        "{} is not running.".format(SCRIPT_NAME))
+
+                if is_valid(ysql_user) or is_valid(ysql_password) or is_valid(ysql_db):
+                    ysql_proxy = YsqlProxy(ip=self.advertise_ip(),
+                                    port=self.configs.saved_data.get("ysql_port"),
+                                    use_env_credentials=False)
+                    if retry_op(ysql_proxy.is_ysql_up):
```

Comment:
in the else case, let's print a log message so we can track it

Reply:
If the retry fails, then it exits the script with the error and logs all the events in the log file. We don't need to provide explicitly else condition here.